src/ThundraLogger.ts

```diff
@@ -2,39 +2,35 @@ import ConfigProvider from './config/ConfigProvider';
 import ConfigNames from './config/ConfigNames';
 
 class ThundraLogger {
-    static instance: ThundraLogger;
-    debugEnabled: boolean;
 
-    constructor() {
-        this.debugEnabled = ConfigProvider.get<boolean>(ConfigNames.THUNDRA_DEBUG_ENABLE);
-        ThundraLogger.instance = this;
-    }
+    static debugEnabled: boolean;
 
-    static getInstance(): ThundraLogger {
```

Comment:
I didn't see where this constructor is called, so if this is the case, I think `static debugEnabled: boolean` field is always equal to the default value `undefined`.